app/models/declaration/ExportItem.scala

```diff
@@ -31,6 +31,7 @@ case class ExportItem(
   itemType: Option[ItemType] = None,
   commodityDetails: Option[CommodityDetails] = None,
   dangerousGoodsCode: Option[UNDangerousGoodsCode] = None,
+  cusCode: Option[CUSCode] = None,
```

Comment:
Could cusCode not just be an `Option[String]`?

Reply:
It could.  So could UNDangerousCode.   CommodityDetails could be a Tuple.   I thought we wanted a case-class to encapsulate the values on each page/form?

Reply:
Only if it helps, otherwise it's a little unneccessary

Reply:
If CommodityDetails has two fields then thats fine

Reply:
Isn't it better to be consistent?

Reply:
Two fields should be in one class, for easier managing this. One field IMO can be in class, but in practice you can build form without model. I'm using this for DestinationCountries. So I stayed with backend model, but I'm using just `Form[String]` to manage form. Here CUSCode can be used like String, so it's okay, but `UNDangerousCode. CommodityDetails could be a Tuple` I'll leave it as it is. Or if `UNDangerousCode` can just one String or `CommodityDetails` can one String we can change it to not using model. But if you have place in model where you need to put `Tuple` I'll prefer to stay with class.

Reply:
Tuple wasn't a serious suggestion!  :-) .  I'm going to stick with what I've got.   As mentioned, it's more consistent if a page has a Form[Model] even if that Model could be a String.